src/components/Layout/Header/NavBar/SubmenuHeader.tsx

```diff
@@ -8,9 +8,10 @@ type ExpandedMenuItemProps = {
   title?: string
   description?: string
   alert?: string
+  onBackClick?: () => void
 }
 
-export const SubmenuHeader = ({ title, description }: ExpandedMenuItemProps) => {
+export const SubmenuHeader = ({ title, description, onBackClick }: ExpandedMenuItemProps) => {
   const { handleBackClick } = useMenuRoutes()
```

Comment:
Now that we effectively have two `handleBackClick`s, rename this one to `handleBackClickDefault` on destructure.

```suggestion
  const { handleBackClick: handleBackClickDefault } = useMenuRoutes()
```